### PR TITLE
[ALLUXIO-1586] Add test coverage for OSSOutputStream.java

### PR DESCRIPTION
**Pull request by albericliu**
Add the test coverage for OSSOutputStream.java. Need help to review it.

### CODE REVIEW

**Comment by albericliu** [comment]
help me to review the code

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by yupeng9** [comment]
Thanks. LGTM. @luoli523 can you take a look?

File: underfs/oss/src/test/java/alluxio/underfs/oss/OSSOutputStreamTest.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+   */
+  @Test
+  @PrepareForTest(OSSOutputStream.class)
+  public void testWrite3() throws Exception {
+    PowerMockito.whenNew(BufferedOutputStream.class)
+        .withArguments(Mockito.any(DigestOutputStream.class)).thenReturn(mLocalOutputStream);
+    PowerMockito.whenNew(BufferedOutputStream.class)
+        .withArguments(Mockito.any(FileOutputStream.class)).thenReturn(mLocalOutputStream);
+    OSSOutputStream stream = new OSSOutputStream("testBucketName", "testKey", mOssClient);
+    byte[] b = new byte[1];
```
Comment (luoli523) [inline underfs/oss/src/test/java/alluxio/underfs/oss/OSSOutputStreamTest.java]: make sure this bytes array more than just one byte?

File: underfs/oss/src/test/java/alluxio/underfs/oss/OSSOutputStreamTest.java [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  @Test
+  @PrepareForTest(OSSOutputStream.class)
+  public void testWrite2() throws Exception {
+    PowerMockito.whenNew(BufferedOutputStream.class)
+        .withArguments(Mockito.any(DigestOutputStream.class)).thenReturn(mLocalOutputStream);
+    PowerMockito.whenNew(BufferedOutputStream.class)
+        .withArguments(Mockito.any(FileOutputStream.class)).thenReturn(mLocalOutputStream);
+    OSSOutputStream stream = new OSSOutputStream("testBucketName", "testKey", mOssClient);
+    byte[] b = new byte[1];
+    stream.write(b, 1, 1);
```
Comment (luoli523) [inline underfs/oss/src/test/java/alluxio/underfs/oss/OSSOutputStreamTest.java]: make sure this bytes array more than just one byte?

**Comment by albericliu** [comment]
@luoli523 what is your mean, do I need to add one more test case: verify the write a bytes array which more than one byte?

what should I do? modify the code and create a new pull request?

**Comment by albericliu** [comment]
@luoli523 need your replay, or I close this comment. thanks

**Comment by albericliu** [comment]
@yupeng9  if no one replay, I will close this.

**Comment by luoli523** [comment]
@albericliu  what I mean is to make sure the write(byte[]) and write(byte[],int,int)  are ok within the case when the byte[] contains more than one byte. But that's not such a big deal. Ok with me if just leave it like this.

**Comment by albericliu** [comment]
@luoli523  thanks for your reply, And I am also from china, my webchat is cnuliuwei, so next time we can enter Chinese, my english is not good. So I close this.

**Pull request closed without merging**